static/src/js/components/TemplateList/__tests__/TemplateList.test.jsx

```diff
@@ -9,7 +9,7 @@ import errorLogger from '@/js/utils/errorLogger'
 import getTemplates from '@/js/utils/getTemplates'
 import deleteTemplate from '@/js/utils/deleteTemplate'
 import NotificationsContext from '@/js/context/NotificationsContext'
-import MMT_COOKIE from '@/js/constants/mmtCookie'
+import MMT_COOKIE from 'sharedConstants/mmtCookie'
```

Comment:
does `@/js/constants/mmtCookie` not work ?

Reply:
Correct. That's only for files within static/src. I had to create a new route.

Reply:
This variable was needed for files within serverless/src as well and Matthew said that we need to keep the static/src and serverless/src separate (no shared contestants or utils). To solve this, I needed to cretae a new folder called sharedConstants which is similar to sharedUtils.